webui/react/src/components/ArchiveSelectFilter.tsx

```diff
@@ -0,0 +1,36 @@
+import { Select } from 'antd';
+import { SelectValue } from 'antd/es/select';
+import React, { useCallback } from 'react';
+
+import SelectFilter from 'components/SelectFilter';
+
+const { Option } = Select;
+
+interface Props {
+  onChange?: (value: string) => void;
+  value?: string;
+}
+
+const ArchiveSelectFilter: React.FC<Props> = ({
+  onChange,
+  value,
+}: Props) => {
+  const handleSelect = useCallback((newValue: SelectValue) => {
+    if (!onChange) return;
+    onChange(newValue.toString());
+  }, [ onChange ]);
+  return (
+    <SelectFilter
+      dropdownMatchSelectWidth={130}
+      label="Show"
+      style={{ minWidth: 100 }}
+      value={value}
+      onSelect={handleSelect}>
+      <Option key="All" value="All">All</Option>
```

Comment:
nit: there is an ALL_VALUE constant in types module you could use here.

Also for the option values we try to stick to lowercase (it shows up the parameter lists if you're sharing a url)